boilerplate/app/navigators/DemoNavigator.tsx

```diff
@@ -5,16 +5,14 @@ import { TextStyle, ViewStyle } from "react-native"
 import { useSafeAreaInsets } from "react-native-safe-area-context"
 import { Icon } from "../components"
 import { translate } from "../i18n"
-import { DemoComponentsScreen, DemoDebugScreen } from "../screens"
-import { DemoPodcastListScreen } from "../screens/DemoPodcastListScreen"
+import { DemoCommunityScreen, DemoComponentsScreen, DemoDebugScreen } from "../screens"
 import { colors, typography } from "../theme"
 import { AppStackParamList, AppStackScreenProps } from "./AppNavigator"
 
 export type DemoTabParamList = {
   DemoComponents: undefined
   DemoDebug: undefined
-  DemoPodcastList: undefined
-  DemoContributing: undefined
```

Comment:
@kateinkim what happened here? We want the podcast screen

Reply:
Did I delete this? I didn't have any merge conflict or anything. I don't know what's going on. My recent pull doesn't have these either.